dqc_us_rules/dqc_us_0004.py

```diff
@@ -11,24 +11,43 @@
 _CODE_NAME = 'DQC.US.0004'
 _RULE_VERSION = '1.0'
 
+
 def assets_eq_liability_equity(val):
     """
     Assets equals Liabilities and Stockholders Equity
+
+    :param val: val to look at the modelXbrl for
+    :type val: val
+    """
+    for fact in _assets_eq_liability_equity(val.modelXbrl):
+        fact_assets, fact_liabilities = fact
+        val.modelXbrl.error(
+            '{}.16'.format(_CODE_NAME),
+            messages.get_message(_CODE_NAME),
+            modelObject=[fact_assets, fact_liabilities],
+            ruleVersion=_RULE_VERSION
+        )
+
+
+def _assets_eq_liability_equity(model_xbrl):
     """
-    for fact_assets, fact_liabilities in _assets_eq_liability_equity(val.modelXbrl):
-        val.modelXbrl.error('{}.16'.format(_CODE_NAME), messages.get_message(_CODE_NAME),
-                            modelObject=[fact_assets, fact_liabilities], ruleVersion=_RULE_VERSION)
+    Yeilds fact assets and fact liablities as long as it is able to
 
+    :param model_xbrl: modelXbrl to check name concepts of
+    :type model_xbrl: ModelXbrl
+    """
+    assets_concept = model_xbrl.nameConcepts[_ASSETS_CONCEPT][0] \
```

Comment:
Not sure if these '\' line breaks are needed.  Checked the style guide, and I don't think they are.